electrum/plugins/revealer/qt.py

```diff
@@ -28,20 +28,20 @@
 
 from .hmac_drbg import DRBG
 
+from btchip.btchipComm import getDongle
```

Comment:
This import will fail if the user does not have btchip-python installed, e.g. on Linux. This would mean they cannot use the Revealer plugin at all.

You should import inside a `try ... except`. It seems atm `getDongle` is only used in one place, so the import could be moved there. (in general it's preferred to do imports at the module level at the top of the file, but conditional imports are an exception I would say)

Reply:
Thanks! and sorry, the ledger support commit is not ready and shouldn't be in the PR at this stage. follow up commits coming.

Reply:
then if you want the other stuff merged faster, you could force-push remove the ledger commit